vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -67,11 +68,39 @@ class HomeActivityViewModel @AssistedInject constructor(
     }
 
     private var checkBootstrap = false
+    private var onceTrusted = false
```

Comment:
Does it work if I kill the app when this occurs? I do not think the `true -> false` scenario will occur again in this case.